src/components/UI/DropdownMenu.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import "./DropdownMenu.css";
+import { ReactComponent as SortIcon } from "../../assets/images/icons/ic_sort.svg";
+
+function DropdownMenu({ onSortSelection }) {
+  const [isDropdownVisible, setIsDropdownVisible] = useState(false);
+
+  const toggleDropdown = () => {
+    setIsDropdownVisible(!isDropdownVisible);
+  };
+
+  return (
+    <div className="sortButtonWrapper">
+      <button className="sortDropdownTriggerButton" onClick={toggleDropdown}>
+        <SortIcon />
+      </button>
+
+      {isDropdownVisible && (
+        <div className="dropdownMenu">
+          <div
+            className="dropdownItem"
+            onClick={() => {
+              onSortSelection("recent");
+              setIsDropdownVisible(false);
+            }}
+          >
+            최신순
+          </div>
+          <div
+            className="dropdownItem"
+            onClick={() => {
+              onSortSelection("favorite");
+              setIsDropdownVisible(false);
+            }}
```

Comment:
렌더부에서 넣어줄 값이 없는 경우에는 해당 값을 따로 함수로 빼서 넣어주시는 걸 추천드려요!

```
const handleItemClick = () => {
     onSortSelection("favorite");
     setIsDropdownVisible(false);
}
```

```suggestion
            onClick={handleItemClick}
```